reset-state

```diff
@@ -60,8 +65,10 @@ set -x
 rm -vrf /snap/*
 set +x
 
-echo
-echo "Starting snapd"
-set -x
-systemctl start snapd
-set +x
+if [ "$snapd_was_active" = "yes" ]; then
+    echo
+    echo "Starting snapd"
+    set -x
+    systemctl start snapd.socket
```

Comment:
(`snapd` is short for `snapd.service`; I spell it out in the script to make it clearer)